### PR TITLE
About us section

### PR DESCRIPTION
**Pull request by Rishav9911**
added the about us section closing issue #31

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
       "version": "1.0.0",
```
Comment (Jagroop2001) [inline package-lock.json:9]: Remove package-lock.json file

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
```
Comment (Jagroop2001) [inline package.json:13]: Remove formatting here

**Comment by Rishav9911** [comment]
I have made the above asked changes, kindly review the PR @Jagroop2001 . Thank you.

**Comment by Jagroop2001** [comment]
@Rishav9911 , please remove `package-lock.json`n file

**Comment by Jagroop2001** [comment]
After that it's ready to merge

**Pull request merged**